packages/mobile/src/walletConnect/saga.ts

```diff
@@ -42,3 +44,9 @@ export function* initialiseWalletConnect(uri: string, origin: WalletConnectPairi
       throw new Error(`Unsupported WalletConnect version '${version}'`)
   }
 }
+
+export function* showWalletConnectionSuccessMessage(dappName: string) {
+  const dappSessionActive = yield select(dappSessionActiveSelector)
+  const successMessage = dappSessionActive ? 'inAppConnectionSuccess' : 'connectionSuccess'
+  yield put(showMessage(i18n.t(successMessage, { dappName })))
```

Comment:
nit: I'm still thinking we'd want to extract strings from the source at some point. This would make it more difficult since it's happening at runtime. Here's an alternative:

```suggestion
  const successMessage = dappSessionActive ? i18n.t('inAppConnectionSuccess', { dappName }) : i18n.t('connectionSuccess', { dappName })
  yield put(showMessage(successMessage))
```

Reply:
oh right, yes done :)